lib/discovery.js

```diff
@@ -501,4 +343,12 @@ function mixinDiscovery(DB2, db2) {
     // }
     // return undefined;
   };
+
+  DB2.prototype.setDefaultOptions = function(options) {
```

Comment:
@raymondfeng sounds good. In the base connector the function will still be an empty one, but yes then we don't have to override here.